# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/xul/nsImageBoxFrame.cpp
# Commit: a261bb633790
# Full Hash: a261bb633790ddca9cc8ce83adca0a1bb36114a6
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2022-02-23 09:38:00
# Regressor Bug: 1714584
# File Overlap Count: 2
# Description:
#   Bug 1714584 - Part 1: Decouple nsDisplayList internal list from nsDisplayItems r=mstange
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138152
# ==============================================================================

diff -r 635f402efb1b -r a261bb633790 layout/xul/nsImageBoxFrame.cpp
--- a/layout/xul/nsImageBoxFrame.cpp	Tue Feb 22 23:42:08 2022 +0000
+++ b/layout/xul/nsImageBoxFrame.cpp	Tue Feb 22 23:42:18 2022 +0000
@@ -330,7 +330,7 @@
   DisplayListClipState::AutoClipContainingBlockDescendantsToContentBox clip(
       aBuilder, this, clipFlags);
 
-  nsDisplayList list;
+  nsDisplayList list(aBuilder);
   list.AppendNewToTop<nsDisplayXULImage>(aBuilder, this);
 
   CreateOwnLayerIfNeeded(aBuilder, &list,